base_classes/NXoptical_window.nxdl.xml

```diff
@@ -0,0 +1,132 @@
+        etc.) is enclosed in a cell, which has windows both in the
+        direction of the source (entry window) and the detector (exit
+        window) (looking from the sample).
+        
+        The windows also add a phase shift to the light altering the
+        measured signal. This shift has to be corrected based on measuring
+        a known sample (reference sample) or the actual sample of interest
+        in the environmental cell. State if a window correction has been
+        performed in 'window_effects_corrected'. Reference measurements should be
+        considered as a separate experiment (with a separate NeXus file), and
+        the reference data shall be :ref:`linked &lt;Design-Links&gt;` in
+        ``reference_data_link``.
+        
+        The window is considered to be a part of the sample stage but also
+        beam path. Hence, its position within the beam path should be
+        defined by the 'depends_on' field.
+    </doc>
+    <group type="NXentry">
+        <field name="window_effects_corrected" type="NX_BOOLEAN">
+            <doc>
+                Was a window correction performed? If so, describe the window
+                correction procedure in ``window_correction/procedure``.
+            </doc>
+        </field>
+        <field name="window_effects_type">
+            <doc>
+                Type of effects due to this window on the measurement.
+            </doc>
+            <enumeration>
+                <item value="interference effects"/>
+                <item value="light absorption"/>
+                <item value="light scattering"/>
+                <item value="other"/>
+            </enumeration>
+        </field>
+        <group name="window_correction" type="NXprocess">
+            <doc>
+                Was a window correction performed? If 'True' describe the
+                window correction procedure in ''
+            </doc>
+            <field name="procedure">
+                <doc>
+                    Describe when (before or after the main measurement + time
+                    stamp in 'date') and how the window effects have been
+                    corrected, i.e. either mathematically or by performing a
+                    reference measurement. In the latter case, provide the link to
+                    to the reference data in ``reference_data_file``.
+                </doc>
+            </field>
+            <field name="reference_data_link" type="NX_NUMBER">
+                <doc>
+                    :ref:`External link &lt;Design-Links&gt;` to the data field in the NeXus file which describes                   
+                    the reference data if a reference measurement for window correction 
+                    was performed.
+
+                    Ideally, the reference measurement was performed on a reference
+                    sample and on the same sample, and using the same conditions as
```

Comment:
Sentence is garbled

Reply:
There is something that should be fixed here:
This is currently on the fairmat-2024-optical-spectroscopy branch
```
Ideally, the reference measurement was performed on on the same sample, using the same conditions as for the actual measurement, with
and, if possible, without windows. It should have been conducted as close in time to the actual measurement as possible.
```

"on on" > "on"
"with and, if possible, without windows" > so both cases with windows and without ?